tests/FINDOLOGIC/Export/Tests/CSVSerializationTest.php

```diff
@@ -53,6 +55,11 @@ public function tearDown(): void
             // Cleanup file after tests have created it.
             unlink(self::CSV_PATH);
         }
+
+        if (file_exists(self::ALTERNATE_CSV_PATH)) {
+            // Cleanup file after tests have created it.
+            unlink(self::ALTERNATE_CSV_PATH);
```

Comment:
:wrench: Repetitive code. Can also be written somewhat like this:

```php
foreach ([self::CSV_PATH, self::ALTERNATIVE_CSV_PATH] as $path) {
    if (file_exists($path)) {
        unlink($path);
    }
}
```

Reply:
Removed this.